internal/pkg/cli/env_init.go

```diff
@@ -545,6 +567,40 @@ func (o *initEnvOpts) askAdjustResources() error {
 	return nil
 }
 
+func (o *initEnvOpts) askAZs() ([]string, error) {
+	if o.adjustVPC.AZs != nil {
+		return o.adjustVPC.AZs, nil
+	}
+	if o.ec2Client == nil {
+		o.ec2Client = ec2.New(o.sess)
+	}
+	azs, err := o.ec2Client.ListAZs()
+	if err != nil {
+		return nil, fmt.Errorf("list availability zones for region %s: %v", aws.StringValue(o.sess.Config.Region), err)
```

Comment:
It's really a tiny difference, when we use `%w` that means that the underlying error is exposed part of the `cli` pkg's interface. Since we don't expect any client to check the underlying ec2 error, I've just marked it as `%v`. It doesn't really matter for us :D